src/libstd/ffi/os_str.rs

```diff
@@ -277,6 +277,12 @@ impl OsStr {
         self.to_bytes().and_then(|b| CString::new(b).ok())
     }
 
+    /// Checks if the string is empty.
+    #[unstable(feature = "os_extras", reason = "recently added", issue = "30259")]
```

Comment:
Should decide between keeping #30259 as a tracking issue for this unstable feature or closing it. I’d rather repurpose the issue in question to be a tracking issue.

Also, perhaps a more self-explanatory feature name would be better? Something like `path_is_empty`?